sbi/simulators/simutils.py

```diff
@@ -5,14 +5,62 @@
 import torch
 from torch import Tensor
 from tqdm.auto import tqdm
+from sbi.simulators.mp_simulator import simulate_mp
+
+
+def simulate_mp_in_batches(
```

Comment:
rename to `mp_simulate_in_batches` or `mp_simulate_in_batches` as it does what `simulate_in_batches` does, but with MP?